pages/analytics_utils.py

```
@@ -0,0 +1,1541 @@
+        
+        # Get priority data if available
+        priority_score = float(row.get('priority_score', 0) or 0)
+        criticality = str(row.get('criticality', 'N/A'))
+        relevance_class = str(row.get('relevance_class', 'N/A'))
+        annual_impact = float(row.get('annual_impact', 0) or 0)
+        
+        # Get additional timeline data from merged Excel
+        estoque_cobertura = float(row.get('Estoque Cobertura', row.get('Estoque_Cobertura', 0)) or 0)
+        qtde_embarque = float(row.get('Qtde Embarque', row.get('Qtde_Embarque', 0)) or 0)
+        compras_ate_30_dias = float(row.get('Compras Até 30 Dias', row.get('Compras_Ate_30_Dias', 0)) or 0)
+        previsao = float(row.get('Previsão', row.get('Previsao', 0)) or 0)
```

Ruff (0.11.9)

Local variable `previsao` is assigned to but never used

Remove assignment to unused variable `previsao`

(F841)